resources/views/settings/subscription/change.blade.php

```diff
@@ -1,3 +1,6 @@
+@php
+    $upgrade = $user->upgradePrice($period, $tier);
```

Comment:
This should be prepared and sent in the controller. It also should be in the Subscription service rather than on the user, otherwise we're loading that function on every single page into memory when we only need it in one place :)